### DISCUSSION TITLE
Daily 21/04

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#153] [Daily Sprint]
- Que hice ayer?
- Que haré hoy?
- Dificultades o Impedimentos.

**Comment by NataliaAlvarezIspc** [comment]
Ayer completé todas las tareas de las historias de usuario, hay que repasarlas para ver si están correctas. También subí la planilla de presentación de proyecto a la plataforma y a la documentación en el repo. Estuve haciendo algunas corecciónes menores en la wiki y en documentación. Avancé un poco con Angular, aunque no pude terminarlo. A la tarde participé de la clase sincrónica.
Hoy voy a terminar con lo de Angular y quiero trasladar lo aprendido a nuestro proyecto.

**Comment by BraianTroncoso** [comment]
Hoy me puse a ver un curso de Angular de varias horas y recien vi 3hs de material(pausando y tomando notas) para ir ya preparandome para la parte frontEnd de la SPA que se viene, espero hoy poder ver un poco mas despues de que termine de trabajar.

**Comment by Brian-Almada** [comment]
Ayer no pude avanzar en nada. Ahora estoy con lo de Angular.

**Comment by rpgrca** [comment]
Tal como lo esperaba hoy no pude hacer mucho, tuve dentista a la mañana y ecografía a la tarde. Voy a hacer un repaso de Angular el finde para empezar con el Sprint 1.
